keyboards/clueboard/2x1800/2019/info.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
This file needs to be rebuilt to support the additional layouts and position of the 2019 version.